src/detect-engine-sigorder.c

```diff
@@ -723,6 +723,50 @@ static int SCSigOrderByPriorityCompare(SCSigSignatureWrapper *sw1,
     return 0;
 }
 
+static int SCSigOrderByIId(SCSigSignatureWrapper *sw1, SCSigSignatureWrapper *sw2)
```

Comment:
pondering renaming Signature::num to Signature::iid, which is where this name comes from. Should probably rename to something clearer

Reply:
Are Signature nums unique? They are, right? Could it be Signature::uid?